onadata/libs/serializers/dataview_serializer.py

```diff
@@ -174,7 +175,7 @@ def validate(self, attrs):
 
         return super(DataViewSerializer, self).validate(attrs)
 
-    def get_count(self, obj):
+    def get_count(self, obj) -> Optional[int]:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5558448)